src/components/Files/FileManagerModal.tsx

```diff
@@ -323,193 +308,177 @@ export const FileManagerModal = ({
   }, [dispatch, selectedFilesIds]);
 
   return (
-    <FloatingPortal id="theme-main">
-      {isOpen && (
-        <FloatingOverlay
-          lockScroll
-          className="z-50 flex items-center justify-center bg-blackout p-3"
-          data-floating-overlay
-        >
-          <FloatingFocusManager context={context}>
-            <div
-              className="relative flex max-h-full flex-col gap-4 rounded bg-layer-3 md:w-[525px]"
-              ref={refs.setFloating}
-              {...getFloatingProps()}
-            >
+    <Modal
+      portalId="theme-main"
+      isOpen={isOpen}
+      onClose={() => onClose(false)}
+      dataQa="file-manager-mpdal"
+      containerClassName="relative flex max-h-full flex-col gap-4 rounded bg-layer-3 md:w-[525px]"
```

Comment:
1) there was `const dismiss = useDismiss(context, { outsidePressEvent: 'mousedown' });`. I suppose we need to use `dismissProps={{ outsidePressEvent: 'mousedown' }}`

2) `relative max-x-full rounded bg-layer-3` already in `Modal` an can be removed